tailscale/client_test.go

```diff
@@ -756,3 +756,52 @@ func TestErrorData(t *testing.T) {
 		assert.Empty(t, tailscale.ErrorData(io.EOF))
 	})
 }
+
+func TestClient_ValidateACL(t *testing.T) {
+	t.Parallel()
+
+	client, server := NewTestHarness(t)
+
+	acl := tailscale.ACL{
+		ACLs: []tailscale.ACLEntry{
+			{
+				Action: "accept",
+				Ports:  []string{"*:*"},
+				Users:  []string{"*"},
+			},
+		},
+		TagOwners: map[string][]string{
+			"tag:example": {"group:example"},
+		},
+		Hosts: map[string]string{
+			"example-host-1": "100.100.100.100",
```

Comment:
Because 100.100.100.100 is special, there is the chance that future changes in ACL handling might treat it differently. Recommend choosing a different IP address for the test.

Reply:
Will update these test values, as it stands the tests just call a mocked HTTP API, so I wasn't being particularly picky about values.